_ml-commons-plugin/conversational-search.md

```diff
@@ -15,7 +15,7 @@ Conversational search is an experimental machine learning (ML) feature that enab
 Currently, conversational search uses two systems to synthesize documents:
 
 - [Conversation memory](#conversation-memory)
-- [Retreival Augmented Generation (RAG) pipeline](#rag-pipeline)
+- [Retrieval Augmented Generation (RAG) pipeline](#rag-pipeline)
```

Comment:
Ha!  How did we miss this one?!  We need a spell checker!

Reply:
It happens from time to time!